### PR TITLE
Add test for multiple sriov network attachments (25834)

### PR DESCRIPTION
**Pull request by mmirecki**
Added test for multiple sriov network attachments
Moved the ipv6 test to operator_test.go, to enable code reuse between the two tests.

### CODE REVIEW

File: conformance/operator_test.go [hunk 1]
```diff
@@ -700,6 +702,61 @@ var _ = Describe("operator", func() {
...
+					pod := createTestPod([]string{"/bin/bash", "-c", "--"},
+						[]string{"while true; do sleep 300000; done;"},
+						testNode, sriovNetworkName, []string{sriovNetworkName, sriovNetworkName})
+					nics, err := network.GetSriovNics(pod, "net")
+					Expect(err).ToNot(HaveOccurred())
+					Expect(len(nics)).To(Equal(2), "No sriov network interfaces found.")
+
+				})
+			})
+			FContext("IPv6 configured secondary interfaces on pods", func() {
```
Comment (fedepaol) [inline conformance/operator_test.go]: Focused test, can you remove it?

File: pkg/util/network/network.go [hunk 2]
```diff
@@ -54,6 +55,21 @@ func CreateSriovPolicy(clientSet *testclient.ClientSet, generatedName string, op
 	return err
 }
 
+func GetSriovNics(pod *k8sv1.Pod, ifcPrefix string) ([]string, error) {
```
Comment (fedepaol) [inline pkg/util/network/network.go]: Can you rename this to "GetNicsByPrefix"?
This does not make any logic about those interfaces being sriov

**Review by fedepaol — COMMENTED** [review]
Just a couple of comments, then looks good

File: pkg/util/network/network.go (hunk 2 above)
Comment (mmirecki) [reply pkg/util/network/network.go]: done

**Pull request merged**